pushmsixscript/docker.d/worker.yml

```
@@ -11,6 +11,7 @@ application_ids:
   then:
     beta: { "$eval": "BETA_APPLICATION_ID" }
     release: { "$eval": "RELEASE_APPLICATION_ID" }
+    esr: { "$eval": "ESR_APPLICATION_ID" }
```

```suggestion
    esr:
      $if: 'COT_PRODUCT == "thunderbird"
      then: { "$eval": "ESR_APPLICATION_ID" }
      else: null    
```

I think we just need to prevent the key from being calculated for firefox.